circle.yml

```diff
@@ -212,11 +212,11 @@ jobs:
       - *install-aleth
       - *add-package-to-workspace
       - *fetch-tests
-      - *test-binaryen
-      - *test-wabt
-      - *test-wavm
+#      - *test-binaryen
```

Comment:
You should add comment why this is disabled.

Reply:
You already disabled evmc-test earlier and no comment is to be found 😉 

However the reason can be found using git blame + git log as it is in the commit message.